templates/shared/pricing/_openstack-consulting.html

```diff
@@ -30,12 +30,12 @@
     <tr>
       <td>Software Defined Networking options</td>
```

Comment:
In the copy docs, the "One-time price" line is bold. Shouldn't it match?

Reply:
no, I will update the copy doc